### ISSUE TITLE
[Building] module keys/codesign is missing

### ISSUE THREAD

**Issue by corneliusludmann** [keeweb/keeweb#1351]
**Describe the bug**
I try to build keeweb in the [Gitpod Cloud IDE](https://gitpod.io/#https://github.com/corneliusludmann/keeweb). When I run `npm run dev` I get the following error:

```
gitpod /workspace/keeweb $ npm run dev

> keeweb@1.12.3 dev /workspace/keeweb
> grunt dev

Loading "Gruntfile.js" tasks...ERROR
>> Error: Cannot find module '../keys/codesign'
Warning: Task "dev" not found. Use --force to continue.

Aborted due to warnings.
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! keeweb@1.12.3 dev: `grunt dev`
npm ERR! Exit status 3
npm ERR! 
npm ERR! Failed at the keeweb@1.12.3 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2019-11-14T13_07_04_298Z-debug.log
```

A workaround seems to create a dummy codesign module like this:
```shell
$ mkdir ../keys && echo 'module.exports = { identities: { app: "" }, appleId: "", teamId: "" }' > ../keys/codesign.js
```

**To Reproduce**
1. Open keeweb in Gitpod: https://gitpod.io/#https://github.com/keeweb/keeweb (or alternatively clone repo elsewhere should work as well—not tested)
2. run `npm install`
3. run `npm run dev`
4. The error above appears.
5. run `mkdir ../keys && echo 'module.exports = { identities: { app: "" }, appleId: "", teamId: "" }' > ../keys/codesign.js`
6. run `npm run dev` again
7. The error is gone.

**Expected behavior**
`npm run dev` should run without error or the steps how to add codesign module should be documented.

**Environment**
- Gitpod.io default workspace
- My Gitpod config: https://github.com/corneliusludmann/keeweb/blob/2b32f10149b03b079c99c08d5e07e857141c730d/.gitpod.yml

**Comment by antelle** [comment]
Hi! These keys are not required to build the webapp, for now you comment these lines out, I'll make them optional soon.

**Comment by antelle** [comment]
fixed in 9c32293fa8729189d32488e0a2fb740b86bf8408
